main.js

```diff
@@ -1,6 +1,81 @@
 //ჯავასკრიპტის კლასები
 
+// 2
 
+class Rectangle {
+    constructor(height, width) {
+        this.height = height;
+        this.width = width;
+    }
+    area (){
+        console.log(this.height*this.width)
+    }
+    perimeter(){
+        console.log( 2* (this.height+this.width)) 
+    }
+    isSquare(){
+        if(this.height === this.width){
```

Comment:
შეგეძლო `return this.height === this.width` დაგეწერა მხოლოდ. ლოგი არც არ არის საჭირო.